Dashboard Scope based variable info added (#5084)
SumoLogic/sumologic-documentation: docs/dashboards/filter-template-variables.md

* Only dashboards that were viewed in the last 3 days are updated.
* Up to 10,000 log values and 1,000 metric values are displayed.
* Values for template variables are based on the time range of the Dashboard.

> * New panels added to the dashboard will automatically adopt any existing scope-based variables to it.

Author: ```suggestion
* New panels added to the dashboard will automatically adopt any existing Scope-based Variable to it.
```

1. Select **Logs Search** as the **Variable Type**.
1. Click in the **Query** field and begin typing your query. Select valid options from the pop-up list as they appear until your query is complete.
1. The **Key** is the metadata field you want to use as the filter. Once a Key is selected the Preview table will show example values based on the query running for the last 15 minutes.
1. (Optional) **Include the option to select all values (\*)** will be selected by default. This includes a wildcard asterisk (\*) in the available options.
1. (Optional) Enter a **Default Value** for the variable.
1. Click **Create Template Variable** to apply the variable to the dashboard.

### Add a Scope-based Variable 

> With Scope-based Variable, you can define a variable automatically applies to all log queries within the scope of the dashboard or panels. With this you can define the key while creating the scope variable and users can select the appropriate value to implement simple filtering for the dashboard. Additionally with Scope-based Variable, you can select the specific panels to which the scope variable should be applied.

Reviewer: ```suggestion
With Scope-based Variable, you can define a variable that automatically applies to all log queries within the scope of the dashboard or panels. With this you can define the key while creating the scope variable and users can select the appropriate value to implement simple filtering for the dashboard. Additionally with Scope-based Variable, you can select the specific panels to which the scope variable should be applied.
```

> With Scope-based Variable, you can define a variable automatically applies to all log queries within the scope of the dashboard or panels. With this you can define the key while creating the scope variable and users can select the appropriate value to implement simple filtering for the dashboard. Additionally with Scope-based Variable, you can select the specific panels to which the scope variable should be applied.

Reviewer 2: ```suggestion
With a Scope-based Variable, you can define a variable that automatically applies to all log queries within the scope of the dashboard or panels. With this you can define the key while creating the scope variable and users can select the appropriate value to implement simple filtering for the dashboard. Additionally with Scope-based Variable, you can select the specific panels to which the scope variable should be applied.
```

> With Scope-based variables, you can define a variable automatically applies to all log queries within the scope of the dashboard or panels. With this you can define the key while creating the scope variable and users can select the appropriate value to implement simple filtering for the dashboard. Additionally with Scope-based variables, you can select the specific panels to which the scope variable should be applied.

Reviewer 2: I would make the casing consistent for "scope-based variable(s)" throughout the doc

> With Scope-based variables, you can define a variable automatically applies to all log queries within the scope of the dashboard or panels. With this you can define the key while creating the scope variable and users can select the appropriate value to implement simple filtering for the dashboard. Additionally with Scope-based variables, you can select the specific panels to which the scope variable should be applied.

Author: ```suggestion
With Scope-based Variable, you can define a variable automatically applies to all log queries within the scope of the dashboard or panels. With this you can define the key while creating the scope variable and users can select the appropriate value to implement simple filtering for the dashboard. Additionally with Scope-based Variable, you can select the specific panels to which the scope variable should be applied.
```

> With Scope-based Variable, you can define a variable that automatically applies to all log queries within the scope of the dashboard or panels. With this you can define the key while creating the scope variable and users can select the appropriate value to implement simple filtering for the dashboard. Additionally with Scope-based Variable, you can select the specific panels to which the scope variable should be applied.

Author: ```suggestion
With a Scope-based Variable, you can define a variable that automatically applies to all log queries within the scope of the dashboard or panels. By setting a key while creating the scope variable, you can select values to enable simple filtering for the dashboard. Additionally, you can specify which panels the scope variable applies to for more granular control.
```

:::info

> - The fields in scope based variable are independent of dashboard and panel time range.

Author: ```suggestion
- The fields in the Scope-based Variable are independent of dashboard and panel time range.
```

> - Fields in scope based variable will also contain the FER field keys.

Author: ```suggestion
- Fields in the Scope-based Variable will also contain the FER field keys.
```

> - The fields in scope based variable are independent of dashboard and panel time range.
> - Fields in scope based variable will also contain the FER field keys.

Reviewer 2: ```suggestion
- Fields in scope-based variables are independent of dashboard and panel time range.
- Fields in scope-based variables will also contain the FER field keys.
```
